lib/enumerize/activerecord.rb

```diff
@@ -59,5 +65,19 @@ def update_all(updates)
         super(updates)
       end
     end
+
+    class Type < ActiveRecord::Type::Value
```

Comment:
This hard-coded `ActiveRecord::Type::Value` probably causes a break 🐛  in Rails 4 (which doesn't have a serialize method). For instance, `ActiveRecord::Type::Integer#type_cast` won't be used in an integer column.